content/general/concepts/front-end/front-end.md

```diff
@@ -0,0 +1,63 @@
+---
+Title: 'Front-end'
+Description: 'The front-end of a website includes elements that the end user interacts with directly.'
+Subjects:
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Android'
+  - 'Browsers'
+  - 'Debugging'
+  - 'Development'
+  - 'Frameworks'
+  - 'iOS'
+  - 'jQuery'
+  - 'Responsive'
+  - 'Search'
+  - 'Semantic'
+  - 'Technical Interviews'
+CatalogContent:
+  - 'paths/front-end-engineer-career-path'
```

Comment:
let's just do two things for CatalogContent:
learn HTML
and the Frontend Engineer Career Path